CardScan/Classes/ScanBaseViewController.swift

```diff
@@ -440,15 +418,18 @@ import Vision
             return
         }
         
+        let testImgArr: [CGImage?] = self.testingImagesSizeCheck()
```

Comment:
from a stylistic perspective, I'd much rather see you return a tuple and assign named variables. eg:

`let (squareCardImage, fullCardImage) = self.testingImageSizeCheck()`

Reply:
can we name name this function something else? Maybe `testingImagesWithSizeCheck`